src/game/ingame_menu.c

```diff
@@ -2714,49 +2714,54 @@ s16 render_pause_courses_and_castle(void) {
     print_text(GFX_DIMENSIONS_RECT_FROM_RIGHT_EDGE(78)+26, 209-70, SM64AP_HaveCap(8) ? "Y" : "N");
     s16 x = -32;
     s16 y = 170;
-    s16 spacing = 18;
+    s16 spacing = 16;
     print_text(GFX_DIMENSIONS_RECT_FROM_LEFT_EDGE(20), 209-20, "ABILITIES");
     gSPDisplayList(gDisplayListHead++, dl_rgba16_text_begin);
     gDPSetEnvColor(gDisplayListHead++, 255, 255, 255, gDialogTextAlpha);
+
+    if (SM64AP_CanDoubleJump()) {
+        u8 str_double_jump[] = { TEXT_DOUBLE_JUMP };
+        print_generic_string(x, y, str_double_jump);
+    }
     if (SM64AP_CanTripleJump()) {
         u8 str_triple_jump[] = { TEXT_TRIPLE_JUMP };
-        print_generic_string(x, y, str_triple_jump);
+        print_generic_string(x, y - spacing, str_triple_jump);
```

Comment:
Why is the spacing change necessary?

Reply:
Adding Double Jump to the ability list results in the last move being cut off with the old spacing. Spacing of 16 was chosen as it results in roughly the same position for the final move (18*9=162, 16*10=160). Here is a screenshot of what the ability list looks like with this change applied:
![image](https://github.com/user-attachments/assets/6348462b-85cc-4da0-9287-cdc5b4e2709e)

Reply:
Spacing of 17 was also considered, but I ultimately didn't like how close it moves the final ability to the bottom of the screen. Here is what that looks like, for reference:
![image](https://github.com/user-attachments/assets/77b0f986-ca53-4f12-b4c3-1d8cd1f3ba77)

Reply:
Yeah that sounds nice